### PR TITLE
Bug 1152126 - Add jetpack-id package.

### PR DESCRIPTION
**Pull request by Gozala**

### CODE REVIEW

File: lib/jetpack-id/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+language: node_js
+node_js:
+  - 0.8
+  - 0.10
```
Comment (erikvold) [inline lib/jetpack-id/.travis.yml]: I don't think we need this file

File: lib/jetpack-id/LICENSE [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
```
Comment (erikvold) [inline lib/jetpack-id/LICENSE]: @jsantell said he was fine changing the license to MPL so I think we should just do that.

File: lib/jetpack-id/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+getID({ id: "jid1-JtUwP0fsy08AKw" }); // null
+```
+
+## Installing
+
+* `npm install jetpack-id`
+
+## License
+
+MIT License, copyright (c) 2014 Jordan Santell
```
Comment (erikvold) [inline lib/jetpack-id/README.md]: if this is a fork then let's just remove this file.

File: lib/jetpack-id/test/test.index.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+      });
+    });
+
+    it("Returns null, does not crash, when providing non-string properties for `name` and `id`", function () {
+      expect(getID({ id: 5 })).to.be.equal(null);
+      expect(getID({ name: 5 })).to.be.equal(null);
+      expect(getID({ name: {} })).to.be.equal(null);
+    });
+  });
+});
```
Comment (erikvold) [inline lib/jetpack-id/test/test.index.js]: let's leave this file out too.

File: lib/jetpack-id/LICENSE (hunk 2 above)
Comment (jsantell) [inline lib/jetpack-id/LICENSE]: @erikvold changing to mpl 2.0 now, get jetpack-id 1.0.0) -- but generally the LICENSE files should remain

**Pull request merged**